subiquity/ui/views/error.py

```diff
@@ -257,8 +257,14 @@ def _pile_elements(self):
 
         self.spinner.stop()
 
-        if self.error_ref.state == ErrorReportState.DONE:
-            assert self.report
+        state = self.error_ref.state
+        if state == ErrorReportState.DONE and self.report is None:
+            # It is possible that the report has finished loading server-side
+            # (and therefore the API returns ErrorReportState.DONE) but hasn't
+            # been loaded locally.
```

Comment:
Mmm this feels like something that has been around since the client/server split. Does the server side every load the reports now? (i.e. can the server ever return LOADING here? it feels a bit inappropriate to have in the API, somehow).

Reply:
I think the server only responds with `ErrorReportState.INCOMPLETE`, `ErrorReportState.DONE` or `ErrorReportState.ERROR_GENERATING`. 

So I guess

```diff
-            # It is possible that the report has finished loading server-side
-            # It is possible that the report has finished generating server-side
            # (and therefore the API returns ErrorReportState.DONE) but hasn't
            # been loaded locally.
```

is more appropriate.